fail2ban/server/filter.py

```diff
@@ -898,22 +898,22 @@ def textToIp(text, useDns):
 		return ipList
 
 	@staticmethod
-	def cidr(i, n):
-		""" Convert an IP address string with a CIDR mask into a 32-bit
-			integer.
+	def netAddr2Dec(ipstring, cidr):
+		""" Convert a string IPv4 address with CIDR
+		to decimal IP address of first address in block (network address).
 		"""
 		# 32-bit IPv4 address mask
 		MASK = 0xFFFFFFFFL
-		return ~(MASK >> n) & MASK & DNSUtils.addr2bin(i)
+		return ~(MASK >> cidr) & MASK & DNSUtils.addr2dec(ipstring)
 
 	@staticmethod
-	def addr2bin(string):
-		""" Convert a string IPv4 address into an unsigned integer.
+	def addr2dec(ipstring):
```

Comment:
Okay, this comment is in binary too :)

It returns a long, whereas typical conversion to binary would return a string of 0's and 1's - as they would otherwise be represented as a decimal (or long).

My thinking was, do you get back a binary string or an integer? Can that more accurately be described as "bin" or "dec"?

Binary value of IP Address "10.0.0.0" is "00001010 00000000 00000000 00000000".  Hex: 0A:00:00:00 Decimal: 167772160.